src/client/pages/Toerisme/Toerisme.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Page,
+  PageContent,
+  PageHeading,
+  Panel,
+  Heading,
+} from '../../components';
+import classnames from 'classnames';
+import styles from './Toerisme.module.scss';
+import { AppRoutes, ChapterTitles } from '../../../universal/config/index';
+import { ChapterIcon, Linkd } from '../../components';
+
+import { useAppStateGetter } from '../../hooks/useAppState';
+
+export default function Toerisme() {
+  const { TOERISME } = useAppStateGetter();
+  // Wait for design to show multiple instances for now we take the first in array
+  const info = TOERISME?.content;
```

Comment:
de naam `info` zegt niet zoveel over de soort data die je binnenkrijgt. Zelf noem ik dit soort data `xxxItems` in dit geval, `toeristischeVerhuurItems`. Hieruit kun je net iets beter afleiden wat er binnenkomt, nét iets beter :-)